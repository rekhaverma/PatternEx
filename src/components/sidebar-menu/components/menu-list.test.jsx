import React from 'react';
import MenuList from './menu-list.component';

describe('<MenuList />', () => {
  let wrapper;
  let props;

  beforeEach(() => {
    props = {
      privileges: {},
      active: 'test',
      openItem: 'test',
      list: [],
      onClick: jest.fn(),
      onOpen: jest.fn(),
    };
    wrapper = shallow(<MenuList {...props} />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

