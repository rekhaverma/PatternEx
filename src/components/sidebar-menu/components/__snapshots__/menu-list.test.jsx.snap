// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuList /> should render correctly 1`] = `<div />`;
